src/main/java/com/iab/openrtb/response/Bid.java

```diff
@@ -19,7 +19,7 @@
  * memory pressure. In order to avoid unnecessary allocations this class is made mutable (as an exception) i.e. this
  * decision could be seen as a performance optimisation.
  */
-@Builder
+@Builder(toBuilder = true)
```

Comment:
The `toBuilder` operation is too expensive, try to avoid it as much as you can.
For this case we don't need it cos we have `setAdm(...)`